packages/types/src/anoma.ts

```diff
@@ -0,0 +1,31 @@
+import { ChainInfo as Chain } from "@keplr-wallet/types";
+
+export type Transaction = {
+  hash: Uint8Array;
+  tx: Uint8Array;
+};
+
+export type Account = {
+  alias: string;
+  address: string;
+  publicKey: Uint8Array;
+};
+
+export interface Signer {
+  sign(account: Account, tx: Transaction): Promise<Transaction>;
+  accounts: Account[];
+}
+
+export interface Anoma {
```

Comment:
I feel that thing name could benefit of being more specific. It is describing the public API of the extension.